docs/reference/run.md

```diff
@@ -1142,14 +1142,16 @@ volume mounted on the host).
 
 ### USER
 
-The default user within a container is `root` (id = 0), but if the
-developer created additional users, those are accessible too. The
-developer can set a default user to run the first process with the
+The default user within a container is `root` (id = 0), but if the developer
```

Comment:
@kblin Thank you for the contribution. I think the conten can be clarified a bit more.  Also, can you expand on what you mean by the "user doesn't have to exist in the container" --- is this the `USER` or the `-u` value?  If I pass a user that doesn't exist in the container is the user used anyway.

---> https://gist.github.com/moxiegirl/3cb01145b2fd73f7499d

The  `root` (id = 0) is the default user within a container. An image developer
can create additional users. Those users are accessible by name.  When passing a numeric
user ID, the user doesn't have to exist in the container.

A developer can set a default user to run container processes using the
Dockerfile `USER` instruction. When you start a conatiner, you can override the
`USER` instruction by passing the `-u` option.

    -u="": Username or UID

> **Note:** If you pass a numeric uid, it must be in the `0-2147483647` range.